site/profile/manifests/core/rucio.pp

```diff
@@ -1,63 +1,38 @@
 # @summary
 #   Install required rucio packages
 #
-class profile::core::rucio () {
-  include profile::core::letsencrypt
-
-  #  Host FQDN
-  $fqdn = fact('networking.fqdn')
-
-  #  Define XRootD Path
-  $xrootd_path = '/opt/xrootd'
-
-  #  Define Yum Packages
-  $yum_packages = [
-    'gcc-c++',
-    'cmake3',
-    'krb5-devel',
-    'libuuid-devel',
-    'libxml2-devel',
-    'openssl-devel',
-    'systemd-devel',
-    'zlib-devel',
-    'devtoolset-7',
-    'xrootd',
-    'voms',
-  ]
-
-  #  Define PIP Packages
-  $pip_packages = [
-    'wheel',
-    'cryptography',
-    'rucio',
-  ]
-
-  #  Signed Certificate Location
-  $le_root = "/etc/letsencrypt/live/${fqdn}"
-
-  #  Generate and sign certificate
-  letsencrypt::certonly { $fqdn:
-    plugin      => 'dns-route53',
-    manage_cron => true,
+class profile::core::rucio {
+  yumrepo { 'xrootd-stable':
+    descr               => 'XRootD Stable Repository',
+    baseurl             => 'https://xrootd.web.cern.ch/repo/stable/el$releasever/$basearch',
+    skip_if_unavailable => 'true',
+    gpgcheck            => '1',
+    gpgkey              => 'https://xrootd.web.cern.ch/repo/RPM-GPG-KEY.txt',
+    enabled             => '1',
+    target              => '/etc/yum.repo.d/xrootd.repo',
   }
-
-  #  Copy the certificates into /etc/grid-security
-  -> cron::monthly { 'update_cert':
-    command => "/bin/rsync  -a --copy-links  --chown=xrootd:xrootd ${le_root}/cert.pem ${le_root}/chain.pem ${le_root}/fullchain.pem ${le_root}/privkey.pem /etc/grid-security/ /dev/null 2>&1",
-    user    => 'root',
-    hour    => 0,
-    minute  => 0,
-    date    => 1,
+  -> package { 'xrootd':
+    ensure => 'installed',
   }
-
-  #  Install Pip3 Packages
-  package { $pip_packages:
-    ensure   => 'present',
-    provider => 'pip3',
+  file { [
+      '/lib/systemd/system/xrootd@.service',
+      '/lib/systemd/system/cmsd@.service',
```

Comment:
Why would saluser need to be able to edit the service units?

Reply:
Are the service units being installed by a package?  If so, the file resource(s) need to be ordered on the package.

Reply:
They want to use `saluser` as a shared account to manage the service.

Reply:
They are using `saluser` as a service account to manage the xrootd related services. 

These service units are installed by the xrootd package.